reana_commons/openapi_specifications/reana_server.json

```diff
@@ -2479,6 +2480,9 @@
                 "email": {
                   "type": "string"
                 },
+                "reana_server_version": {
+                  "type": "string"
+                },
```

Comment:
Please rebase against latest `maint-0.7`, something like:

```console
$ git checkout maint-0.7
$ git fetch upstream
$ git merge --ff-only upstream/maint-0.7
$ git checkout 144
$ git rebase maint-0.7 -i
```

Reply:
BTW note that you can also catch these locally before pushing a PR set for review in the following manner:

```console
$ reana-dev git-fetch -c .
$ reana-dev git-status -c . --base maint-0.7
reana-commons @ pr-231 (STEMS FROM 7 BEHIND upstream/maint-0.7) @ 6cb9afa openapi: return reana_server_version
``` 

The warnings in red such as the given branch stems from behind upstream etc may be printed if something is not in order.

Reply:
Done 👍